force-app/main/default/classes/AccountValidation.cls

```diff
@@ -0,0 +1,20 @@
+public with sharing class AccountValidation {
+    public static Boolean validateAccount(Account anAccount){
```

Comment:
validate account lo usas en un for, y aqui tienes una query eso seria hacer una query dentro de un for, deberias bulkificar esta funcion